packages/base/src/3dview/mainview.tsx

```diff
@@ -1065,6 +1065,24 @@ export class MainView extends React.Component<IProps, IStates> {
         );
       }
 
+      // Update near and far for orthographic camera if applicable
+      if (this._camera instanceof THREE.OrthographicCamera) {
+        const near = Math.max(this._refLength / 20, 0.01);
+        const far = this._refLength * 20;
+
+        const distance = this._camera.position.distanceTo(
+          this._controls.target
+        );
+        const zoomFactor = 1000 / distance;
+        this._camera.zoom = zoomFactor;
```

Comment:
I'd only change the zoom level if `updateCamera` is true:

```suggestion
        if (updateCamera) {
          this._camera.zoom = zoomFactor;
        }
```

This flag was introduced to prevent camera jumping when simply editing the current scene (e.g. you wouldn't want to zoom-in zoom-out when moving objects around with the transform control).

Though I'd keep near and far setting at all times.

Reply:
Thanks Martin, I didn't remember what `updateCamera` was for.